mmdet/models/dense_heads/yolof_head.py

```diff
@@ -121,8 +121,11 @@ def forward_single(self, x: Tensor) -> Tuple[Tensor, Tensor]:
         # implicit objectness
         objectness = objectness.view(N, -1, 1, H, W)
         normalized_cls_score = cls_score + objectness - torch.log(
-            1. + torch.clamp(cls_score.exp(), max=INF) +
-            torch.clamp(objectness.exp(), max=INF))
+            1. + torch.clamp(
```

Comment:
Thank you very much for your feedback. The formatting of this is not very good, can you simply write a function to beautify it?

Reply:
sounds good.